azure-pipelines.yml

```
@@ -44,13 +44,24 @@ stages:
             pip install -U -r requirements.txt
           displayName: 'Install requirements'
         - script: |
-            pytest
+            pytest --cov=anonlinkclient --junitxml=testResults.xml  --cov-report=xml:coverage.xml
           displayName: 'Tests'
-#        - bash: |
-#            pip install -U pip codecov
-#            report_name="ubuntu18-04_$(PYTHON_VERSION)"
-#            python -m codecov --token $(CODECOV_TOKEN) \
-#              --file coverageReport.xml \
-#              -n ${report_name}
-#          displayName: 'Send coverage to codecov'
-#          condition: succeededOrFailed()
+          timeoutInMinutes: 10  # it should pass in 2.5 minutes, so 10 should be more than enough even if server is busy
+          env:
+            TEST_ENTITY_SERVICE: 'https://testing.es.data61.xyz'
```

Could change this now (although unrelated to the focus of this PR on **improving code coverage**)

```suggestion
            TEST_ENTITY_SERVICE: 'https://anonlink.easd.data61.xyz'
```

---

2 tests failed with new entity service link:

```TimeoutError: Timeout exceeded before run 8c53a85f2a4bf1dd6dabdec64921e3b97dee73e671fc79e9 terminated``` in `test_2_party_upload_and_results`